lib/Backends/CPU/CPUBackend.cpp

```diff
@@ -174,7 +174,7 @@ bool CPUBackend::isOpSupported(Kinded::Kind opKind, ElemKind elementTy) const {
     }
   }
 
-  return true;
+  return elementTy == ElemKind::FloatTy;
```

Comment:
strictly speaking not all fp32 operations are always implemented even if we have glow instr in place. Same goes to the interpreter.